pages/CONTRIBUTING-docs.md

```diff
@@ -0,0 +1,72 @@
+
+## Before you start
+To contribute to the JSON Schema documentation you will need to complete the following steps:
+
+1. [Create a GitHub account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github), if you don't have one already.
+2. Read our [Code of Conduct](https://json-schema.org/overview/code-of-conduct).
+3. [Install the JSON Schema website](https://github.com/json-schema-org/website/blob/main/INSTALLATION.md) on your computer.
+4. Read our style guide. 
+
+## Community engagement
+To stay updated with news about the JSON Schema project and its documentation, engage with our team, ask questions, and discuss ideas, please join our [Slack workspace](https://json-schema.org/slack). We particularly recommend the following channels:
+
+- #documentation: Discussions related to the JSON Schema documentation.
+
+- #website: Discussions related to our website.
+
+- #introductions: Self-introductions and connecting with other members of the community.
+
+- #community-announcements: Events and updates related to the JSON Schema project.
+
+- #anouncements: Important announcement from the JSON Schema organization.
+
+- #stack-overflow: Stack overflow questions tagged with JSON Schema.
+  
+## Best practices
+
+This is a list of best practices the JSON Schema project strives and resources we recommend for additional information:
+
+1. File and branching naming conventions: We use dashes instead of spaces, and lowercase letters to name .md files and branches. For example: _implementers-guide.md_, _web-feat-diataxis_.
+2. Commit messages: To learn how to write good commit messages, see [Step-by-step guide on how to write good commit messages on Git](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/).
+
+## Contribution workflow 
+We value documentation and appreciate efforts to help us improve our documents. There are two ways you can contribute to improve our documentation:
+
+
+1. Review pull requests
+2. Work on an existing issue 
+3. Propose changes 
+
+### Review pull requests
+
+This is a great way to start contributing. Reviewing pull requests helps you get familiar with our workflows, know how other people contribute, and understand the project's needs and goals. 
+
+### Work on an existing issue
+
+To work on an existing issue, go to the [JSON Schema docs board](https://github.com/orgs/json-schema-org/projects/16) and check all the open issues related to documentation. 
+
+If the issue you select has no asignees, you can claim the issue by assigning it to yourself and tag @benjagm in the comments to let us know you'll be working on it. 
+
+If the issue you select has already an asignee, write a comment expressing your interest in collaborating with them. The JSON Schema respects the order in which people claim issues, therefore, you will have to check if the asignee is accepting collaborations.
```

Comment:
assignees - typo in the first line and the second line.
Grammar change in line one.

The correct sentence would be - 

If the issue you select **already has** an **assignee**, write a comment expressing your interest in collaborating with them. The JSON Schema respects the order in which people claim issues, therefore, you will have to check if the **assignee** is accepting collaborations.

Reply:
Thank you for catching this detail